### DISCUSSION TITLE
I can't get the Access Token "The grant type field is required"

### DISCUSSION THREAD

**Discussion by paolocerto** [fattureincloud/api#246] [Q&A] [answered]
This is the error message

PHP Fatal error:  Uncaught GuzzleHttp\\Exception\\ClientException: Client error: `POST https://api-v2.fattureincloud.it/oauth/token` resulted in a `400 Bad Request` response:\n{"error":"invalid_request","error_description":"The grant type field is required."}\n

I followed the instructions and my code is this

$code = $_GET['code'];
$oauth = new \FattureInCloud\OAuth2\OAuth2AuthorizationCodeManager($client_id, $secret, $back);
$obj = $oauth->fetchToken($code);

I  can't see the error.
Can someone help me?
Thank you

**Comment by emajo** [comment, marked as answer]
HI @paolocerto, first of all make sure that you are using the last version of the sdk, then check if your code is correct, we have a working example you can check out [here](https://github.com/fattureincloud/fattureincloud-quickstarts/blob/master/sdk/php/oauth.php).

Let us know if you were able to solve the problem,
thanks,
Emanuele.

**Reply by valmoz** [reply]
Hi @paolocerto 
were you able to solve this issue?

Thanks
Mauro

**Reply by paolocerto** [reply]
Yes, it was the old version of GuzzeHttp
thank you for your help
